index.html

```diff
@@ -1094,25 +1094,26 @@ <h2 class="ui header">Glyphs <em class="fontname" style="display: none;">FontNam
       
       if(t.attr('data-dis') == 0)
       {
+
         glyphs.push(
-          ' ' + (isFirstCharacter ? ' ' : ',') + '{ ' +
+          '  { ' +
                   ('     ' + offset).slice(-5) + ', ' +
                   ('   ' + w).slice(-3) + ', ' +
                   ('   ' + h).slice(-3) + ', ' +
                   ('   ' + parseInt(t.attr('data-adv'))).slice(-3) + ', ' +
                   ('    ' + parseInt(t.attr('data-ow'))).slice(-4) + ', ' +
-                  ('    ' + parseInt(t.attr('data-oh'))).slice(-4) + ' }   ' +
-                  "// '" + char + "'")
+                  ('    ' + parseInt(t.attr('data-oh'))).slice(-4) + ' }' + (isLastCharacter ? ',' : ' ') + '   ' +
```

Comment:
Shouldn't it be `!isLastCharacter` ?

Reply:
Yes, but I've already considered this by swaping `','` and `' '`
So no change needed here.

Reply:
Yes - what I meant is that the variable is not well named (my fault initially) since if it's the last character, we should not put a comma. I think we should change line 1084 and invert the comparison somehow, and invert the characters here (not sure if I'm clear ...)